python/ray/experimental/sgd/util.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
```

Comment:
The `ray` import should be separate and below standard library imports https://github.com/google/styleguide/blob/gh-pages/pyguide.md#313-imports-formatting

This applies in other places also

Reply:
Fixed.